css/style.css

```diff
@@ -14,4 +14,186 @@ a:hover{text-decoration: none;}
 ul li{list-style: none;}
 img{vertical-align: top;}
 h1,h2,h3,h4,h5,h6{font-size:inherit;font-weight: inherit;}
-/*--------------------*/
+/*----------------------------------*/
```

Comment:
the names of the classes are self-explanatory, you can delete all the -------- comments. If you want to separate them somehow, you can break this CSS file into multiple ones